lib/cssToJss.js

```diff
@@ -58,12 +58,9 @@ function toJssRules(cssRules, options) {
   }
 
   function formatProp(prop) {
-    if (options.dashes) {
-      // Capitalize the first character
-      return (typeof prop === 'string' && prop.substring(0, 1) === '-') ?
-        camelCase(prop).charAt(0).toUpperCase() : camelCase(prop)
-    }
-    return prop
+    // Capitalize the first character
+    return options.dashes ? prop : (typeof prop === 'string' && prop.substring(0, 1) === '-') ?
```

Comment:
just looked around, there is a good popular module, lets just use this one: https://github.com/rexxars/hyphenate-style-name

Should be using it everywhere else, it is caching the result, css has only  a finite amount of props, so it will evtl be very fast and not too much for memory

Reply:
ups,  we need exactly the opposite direction

Reply:
I will just merge it for now and revisit it later